Components/button.css

```diff
@@ -0,0 +1,176 @@
+/* BUTTON Component */
+
+.btn {
+  padding: 0.5rem 1rem;
+  margin: 0.25rem;
+  background: transparent;
+  text-align: center;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  font-size: 0.8rem;
+  cursor: pointer;
+  transition: background 0.15s;
+  font-family: inherit;
```

Comment:
font-family is an inherited property, so not required for each

Reply:
Fixing it right now.